pkgs/development/tools/golangci-lint/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "golangci-lint";
-  version = "1.55.2";
+  version = "1.56.1";
 
   src = fetchFromGitHub {
     owner = "golangci";
     repo = "golangci-lint";
     rev = "v${version}";
-    hash = "sha256-DO71wfDmCuziEcsme1g1uNIl3MswA+EkQcYzOYHbG+I=";
+    hash = "sha256-6mwdDi9ltEKpDNa+GPEHiJdQO8csUg6MnuyiKQ02B80";
```

```suggestion
    hash = "sha256-6mwdDi9ltEKpDNa+GPEHiJdQO8csUg6MnuyiKQ02B80=";
```

just checking my knowledge

---

I am surprised this works but we should add the = back if the hash nix gave us had it.

Interesting, nix adds this somehow internally.
```
specified: sha256-6mwdDi9ltEKpDNa+GPEHiJdQO8csUg6MnuyiKQ03B80=
```

---

my understanding is that base64 should be adding this